packages/SwingSet/src/devices/command.js

```diff
@@ -52,7 +48,7 @@ export default function buildCommand(broadcastCallback) {
     }
     if (!responses.has(count)) {
       // maybe just ignore it
-      throw new Error(`unknown response index ${count}`);
+      throw assert.fail(d`unknown response index ${count}`);
     }
```

Comment:
Can you explain the difference here? In this file, I see three cases of 
```
if (X) {
  throw new Error('MSG');
}
```
transformed to
```
assert(X, d'MSG');
```
and one  to 
```
if (X) {
  throw assert.fail(d'MSG');
}
```

Reply:
This one should indeed be as `assert(` rather than as `if.. throw assert.fail(`. Likely the comment caused my regexp searches to miss it. I'll fix it.

Please flag any others like that you see. Thanks.

Reply:
Fixing to
```js
    assert(responses.has(count), X`unknown response index ${count}`);
```

This is SwingSet so @warner @FUDCo ,
What should happen to the
```js
// maybe just ignore it
```
comment? I'm not familiar enough with the code to know what to do with it.

Reply:
I think this is the boundary between swingset's waterken-like commitment domain and the outside-world's internet-like ad-hoc-retry domain, so it's not immediately obvious how likely that error case is (or whether it should qualify as an error at all). I'd recommend retaining the comment, maybe update to "this might not qualify as an error, it needs more thought".

Let's see, this could happen if:

* the ag-solo is loaded, an HTTP frontend sends in a command, the requests is queued but not responded to yet
* the ag-solo is terminated
* the ag-solo is reloaded: `command.js` is re-loaded but `responses` is ephemeral, so it is empty
* the request is processed and the response is returned to the command device
* `deliverResponse` is invoked with a `count` that isn't in the new copy of `responses`

In that case, the HTTP connection is long-gone too, so there's nobody still waiting for the answer (they received a ECONN or 500 or some other "lost the connection before getting an answer" error shortly after the ag-solo was terminated). We need to make sure we don't deliver the answer to the wrong place (if a new connection was made and more requests submitted, we must not re-use the `count` value if that might cause confusion).

This could also happen if the caller (`command-src.js`, in `sendResponse`, which is called by something in the HTTP handler vat) gets confused and sends a bogus `count`. In that case, it's a bug in the calling code. But if it's hit because of a restart, that's not something the calling code could know about. I think that "sometimes a bug, sometimes not" is why I made `command-src.js` catch+log any errors thrown by `sendResponse`, to hide them from callers.

Reply:
Given this extensive analysis, I added both your suggested text and a lint to your message above.